### PR TITLE
Added Content vew tests

### PR DESCRIPTION
**Pull request by sean797**
in preparation for a content_view_filter module

### CODE REVIEW

File: test/test_crud.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     'setting',
     'sync_plan',
     'activation_key',
+    'content_view',
```
Comment (mdellweg) [inline test/test_crud.py]: I thought, this was suppost to be in alphabetical order...

**Comment by mdellweg** [comment]
I have no idea how, but you seem to have managed, to break all tests but job_template.

**Comment by sean797** [comment]
I have no idea either, I've even tried re-recording of them again and Travis still seems unhappy :-1:

**Comment by mdellweg** [comment]
And there it is also no new version of vcrpy that slipped in unnoticed.

**Comment by mdellweg** [comment]
At least, now it's failing my PR too.

**Comment by mdellweg** [comment]
See #138

File: test/test_playbooks/content_view_filter.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (mdellweg) [inline test/test_playbooks/content_view_filter.yml]: Is this the beginning of test driven development?

File: test/vcr_python_wrapper.py [hunk 3]
```diff
@@ -7,23 +7,6 @@
...
-        body2 = json.loads(r2.body.decode('utf8'))
-        if 'search' in body1:
-            body1['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body1['search'])))
-        if 'search' in body2:
-            body2['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body2['search'])))
-        return body1 == body2
-    else:
-        return r1.body == r2.body
-
-
```
Comment (mdellweg) [inline test/vcr_python_wrapper.py]: Did they change the behavior of vcrpy here?

File: test/test_playbooks/content_view.yml [hunk 4]
```diff
@@ -1,19 +1,46 @@
...
+  tasks:
+  - name: 'Load server config'
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/content_view.yml
+    vars:
+      expected_change: true
+  - include: tasks/content_view.yml
+    vars:
+      #repository_state: present
```
Comment (mdellweg) [inline test/test_playbooks/content_view.yml]: I don't think, this is what you meant. ;)

File: test/test_playbooks/content_view_filter.yml (hunk 2 above)
Comment (sean797) [reply test/test_playbooks/content_view_filter.yml]: :man_facepalming:

File: test/vcr_python_wrapper.py (hunk 3 above)
Comment (sean797) [reply test/vcr_python_wrapper.py]: I think it may have been https://github.com/kevin1024/vcrpy/pull/340/files#diff-dc8efcefc2dd30550706019d468fa580R1

**Comment by mdellweg** [comment]
Tanks @sean797

**Pull request merged**